student-library/src/main/java/tds/student/services/abstractions/IResponseService.java

```diff
@@ -32,4 +32,6 @@
   boolean isTestComplete (UUID oppKey) throws ReturnStatusException;
 
   void removeResponse (OpportunityInstance oppInstance, int position, String itemID, String dateCreated) throws ReturnStatusException;
+
+  void markItemForReview(final OpportunityInstance opportunityInstance, final int position, final boolean mark) throws ReturnStatusException;
```

Comment:
Can we add a JavaDoc for this? I know previous maintainers haven't done this but we can slowly chip away at it.